cpp/tests/binaryop/binop-integration-test.cpp

```diff
@@ -2060,13 +2061,13 @@ TYPED_TEST(FixedPointTestBothReps, FixedPointBinaryOpAdd)
   using namespace numeric;
   using decimalXX = TypeParam;
 
-  auto const sz = std::size_t{1000};
+  auto const sz = std::size_t{10};
```

Comment:
We  need to test things larger than thread block, so it seems like 1000 is a better size than 10.

Reply:
Right sorry this was supposed to only temporary for debugging.

Reply:
Done.